docs/Anjay_integrations/Anjay_ST_integration/B-L475E-IOT01A.md

```diff
@@ -14,14 +14,15 @@ Integrate your B-L475E-IOT01A Discovery kit board.
 
 To get the latest binaries, navigate to [Anjay-zephyr-client](https://github.com/AVSystem/Anjay-zephyr-client/releases).
 
-In this case we are going to use the demo_B-L475E-IOT01A1_merged.bin file which after being downloaded must be added to B-L475E-IOT01A board folder.
-
-To flash the board, drag the .bin file to B-L475E-IOT01A device which you can find in **devices and drives** part in **This PC**. You will see the blinking diode on your board. As soon as the diode stops blinking, the flashing has finished.
+To flash the board, open **Windows Explorer**, drag the .bin file to B-L475E-IOT01A device which you can find in **devices and drives** part in **This PC**. You will see the blinking diode on your board. As soon as the diode stops blinking, the flashing has finished.
```

Comment:
Which .bin file would that be @malw4 ? https://github.com/AVSystem/Anjay-freertos-client/releases/download/22.09/Anjay-freertos-client-B-U585I-IOT02A-BG96.bin ?